### ISSUE TITLE
Issues running

### ISSUE THREAD

**Issue by 8bitgentleman** [louisroy/atlas-obscura-roadtrip-planner#1]
This is a really cool idea and I'd love to give it a try but I'm having some trouble getting it to run.  The heroku webapp is just a blank page
<img width="1440" alt="Screen Shot 2019-04-22 at 11 50 24 AM" src="https://user-images.githubusercontent.com/4028391/56509724-c10b0200-64f5-11e9-93cb-91dc7b491400.png">

and trying to run it via npm throws this error [this error](https://gist.github.com/8bitgentleman/e039d38c4e16bef2d75037f97218afb1)

Any ideas on how to get this working? Thanks!

**Comment by louisroy** [comment]
I've been struggling to find a free/consistent/secure geolocator to populate the first text input in the UI. That's what's causing the blank page on Heroku. I'll eventually get around making the UI a bit more resilient to this sort of stuff.

Either way, I've fixed it on Heroku. Let me know how it goes on your end.

As far as running it locally, I'm not sure it'll be compatible with Node 4.x (which is what you have it looks like). I haven't really paid attention to compatibility stuff while building out this prototype. I'd recommending upgrading Node to see if it helps. I'll do some tests and explicitly set the engine version in package.json once I'm done.

If you have any ideas or comments to improve this little project, feel free to do so!

**Comment by 8bitgentleman** [comment]
Thanks for the quick response! Heroku is working now and you were right, updating node fixed the error I was getting. Off the top of my head, after playing around a bit, one improvement could be the ability to share a route via URL. I've constructed a route with several waypoints and would like to share it with others who I'll be traveling with. I could send them a link that included all the waypoints I've added so they can see/edit the route. If I think of anything else I'll let you know. This little app was exactly what I was looking for, thanks for making it!
